### ISSUE TITLE
Sort Filter - 2.4.7 Focus Visible - Focus is not visible on the Sort Filter button

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9994]

**Comment by johnatawnclementawn** [comment]
Same as #9898

**Comment by imogenroberts** [comment]
RETEST: Sort Filter now has a focus indicator which matches the rest of the focusable components on the page.

**Evidence:**
![image](https://github.com/archesproject/arches/assets/77161085/106c3e7d-f9e3-4716-a4c4-66e35c163881)
